coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -32,14 +33,20 @@ def __init__(self, documentation, docstyle_definition,
         :param marker:
             The three-element tuple with marker strings, that identified this
             documentation comment.
-        :param range:
-            The position range of type ``TextRange``.
+        :param position:
+            The starting ``TextPosition`` of the documentation.
         """
         self.documentation = documentation
         self.docstyle_definition = docstyle_definition
         self.indent = indent
         self.marker = marker
-        self.range = range
+        self.position = position
+        self.assemble = None if position is None else self._assemble()
```

Comment:
Btw i liked the idea of `@lru_cache` Im implementing it. It will be good when later part (bear) calls it and it will never get outdated YAY! :wink`